### ISSUE TITLE
Balance goes down

### ISSUE THREAD

**Issue by kbaf** [CPChain/chain#8]
The balance of my wallet is going down with small pieces of cpc.
I am getting rewards for sealing blocks, but in between my balance goes down.
Am i paying for something? TXS fees ?

**Comment by alex031029** [comment]
As a miner, your node calls smart contract constantly to claim campaign, in order to become the proposer. We implement this claiming campaign by send out transaction with 0 value. But calling smart contract itself is not free. Thus, you have to pay a small amount of gas (fees) to the smart contract, which will finally go to the proposer who mined the block with your transactions. However, there is no need to worry about this issue during beta test, since your reward is counted merely on the number of blocks you mined. Other factors are unrelated with it. 

TL;DR: It is the fee for claiming campaign. It is normal.
